spec/rubocop/cop/rails/where_range_spec.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::WhereRange, :config do
+  context 'Ruby <= 2.6', :ruby26 do
+    it 'does not register an offense when using anonymous `>=`' do
+      expect_no_offenses(<<~RUBY)
+        Model.where('column >= ?', value)
+      RUBY
+    end
+  end
+
+  context 'Rails 5.1', :rails51 do
+    it 'does not register an offense when using anonymous `>=`' do
+      expect_no_offenses(<<~RUBY)
+        Model.where('column >= ?', value)
+      RUBY
+    end
+  end
+
+  context 'Ruby >= 2.7', :ruby27 do
+    context 'Rails 6.0', :rails60 do
+      it 'registers an offense and corrects when using anonymous `>=`' do
+        expect_offense(<<~RUBY)
+          Model.where('column >= ?', value)
+                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `where(column: value..)` instead of manually constructing SQL.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          Model.where(column: value..)
```

Comment:
Endless ranges `(start..)` are supported since Ruby 2.6, while beginless ranges `(..end)` are supported since Ruby 2.7. Each should be considered accordingly.

Reply:
Ruby 2.7 is EOL for a long time. Not to mention 2.6. 
Instead, wdyt about dropping the support for it altogether from the gem?

Reply:
The runtime Ruby version (2.7+) and parsing Ruby version (2.0+) are different. This is the parsing Ruby version.

Reply:
Do you mean by the original comment, that I should update the test cases only or the code of the cop too?

Reply:
RSpec supports Ruby 1.8.7
RuboCop “Targets Ruby 2.0+ code analysis”.
RuboCop Rails:
> Rails cops support the following versions:
> Rails 4.2+

Please don’t get me wrong, I always advocate to drop support for EOL versions, even in minor/patch versions. But while those versions are supported, we need to be compatible.

Reply:
I added `minimum_target_ruby_version 2.7`. Do you mean to lower it to `2.6` and handle both cases of endless ranges support differently?

Reply:
I think it’s just perfect. My apologies for my ambiguous language, I meant not to break it for earlier ruby/rails versions.

Reply:
> Do you mean to lower it to `2.6` and handle both cases of endless ranges support differently?

Yes.

Reply:
Updated. Please, take a look.